coopr-docs/docs/source/rest/metrics.rst

```diff
@@ -81,3 +81,149 @@ Example
            "total": 1
        }
    }
+
+
+
+Get Node Usage Metrics
+======================
+
+To get node usage metrics, HTTP GET request to URI:
+::
+
+/metrics/nodes/usage
+
+Only admins are allowed to get node usage metrics. Tenant admins will see metrics for their own nodes,
+whereas the superadmin will get metrics for all tenant nodes.
+
+
+HTTP Responses
+==============
+
+The response will be a JSON Object with start and end time as keys, and the node usage metrics as the values.
+node usage metrics include the key value pairs entered as parameters, followed by the resulting node hour usage
+based on the selected criteria.
+
+.. list-table::
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Status Code
+     - Description
+   * - 200 (OK)
+     - If update was successful
+   * - 403 (FORBIDDEN)
+     - If the user is forbidden from getting node usage metrics.
+
+
+
+
+API Endpoints and Examples
+==========================
+
+
+General usage
+-------------
+
+.. code-block:: bash
+
+ $ curl -H 'Coopr-UserID:admin'
+        -H 'Coopr-ApiKey:<apikey>'
+        -H 'Coopr-TenantID:superadmin'
+        http://<server>:<port>/<version>/metrics/nodes/usage
+
+ $ {"start":1424475097,"end":1424485733,"data":[{"time":1424475097,"value":29035}]}
+
+
+The value at the end of the resulting json here, is the number of node seconds for all nodes combined.
+
+As an example, if you have a 3 node cluster and a 1 node cluster (total 4 nodes), every 60 seconds,
+you will have 240 (4 nodes x 60 seconds) additional node seconds.  The following two lines are examples
+of the json results returned when running a general api node usage call 60 seconds apart:
+
+.. code-block:: bash
+
+ {"start":1424475097,"end":1424486443,"data":[{"time":1424475097,"value":31872}]}
+ {"start":1424475097,"end":1424486503,"data":[{"time":1424475097,"value":32112}]}
+
+
+
+Time Ranges
+-----------
+
+Start and end values are expressed in Unix time (a.k.a. seconds since epoch, or the beginning of Unix time:
+00:00:00 Coordinated Universal Time (UTC), Thursday, 1 January 1970).
+Unix time may be checked on most Unix systems by typing date +%s on the command line.
+
+In the general usage example, in the json response to general node hour usage, we see the start and end values
+returned. If start and end are not specified, the start time is the exact date and time when the very first 
+task (ever) completes. The end date is the current date and time.
+
```

Comment:
With the general usage example, the JSON response to general node hour usage shows the start and end values returned. If the start is not specified, the start time is the exact date and time when the very first 
task (ever) completes. The end date, if not specified, is the current date and time.